content/en/docs/reference/access-authn-authz/node.md

```diff
@@ -34,7 +34,7 @@ Write operations:
 
 Auth-related operations:
 
-* read/write access to the certificationsigningrequests API for TLS bootstrapping
+* read/write access to the [`CertificateSigningRequest`](/docs/reference/access-authn-authz/certificate-signing-requests/) API for TLS bootstrapping
 * the ability to create tokenreviews and subjectaccessreviews for delegated authentication/authorization checks
 
 In future releases, the node authorizer may add or remove permissions to ensure kubelets
```

Comment:
Unrelated cleanup suggestion: we typically [avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future) - we save those statements for blog articles, [k/enhancements](https://github.com/kubernetes/enhancements/), and other loci.